tests/Model/ValueObject/String/Behaviour/MockHasMaxLength.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\String\Behaviour;
+
+class MockHasMaxLength
```

Comment:
Check how `MockNotEmptyString` handles this

Reply:
Looks the same to me, how would you prefer it? Something like "MockNotTooLong" ?

Reply:
I guess it is `MockHasMaxLengthString`